Components/XandYLabelsGraph/index.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import Paragraph from "../Paragraph";
+import InputTypeText from "../InputTypeText";
+import Heading from "../Heading";
+import { StyledInputTypeSubmit } from "../StyledInputTypeSubmit";
+import CenteredDiv from "../CenteredDiv";
+
+export default function XandYLabelsGraph({
+  onXLabelChange,
+  onYLabelChange,
+  hasEnteredYLabel,
+}) {
+  const xKey = "age";
+  const yKey = "boys-height";
+
+  function handleSubmit(event) {
+    event.preventDefault();
+  }
+
+  function handleAxesLabels() {
+    alert("Labels for the x-axis and the y-axis of the graph are assigned");
+  }
+
+  return (
+    <>
+      {xKey != "" && yKey != "" ? (
+        <>
+          <Heading>Step 4: Give labels to axes</Heading>
+          <Paragraph>
+            For the x variable you chose:<br></br>
+            {xKey}
+          </Paragraph>
+          <Paragraph>
+            For the y variable you chose:<br></br>
+            {yKey}
+          </Paragraph>
+          <CenteredDiv>
+            <form onSubmit={handleSubmit}>
+              <InputTypeText
+                idString={"Label for the x-axis"}
+                placeholderString={
+                  "Fill me with name and unit, e.g. Age (years)"
+                }
+                onChange={onXLabelChange}
+              />
+              <br></br>
+              <br></br>
+              <InputTypeText
+                idString={"Label for the y-axis"}
+                placeholderString={
+                  "Fill me with name and unit, e.g. Height (inches)"
+                }
+                onChange={onYLabelChange}
+              />
+              <br></br>
+              <br></br>
+              <StyledInputTypeSubmit
+                value="Next"
+                onClick={handleAxesLabels}
+                disabled={!hasEnteredYLabel}
```

Comment:
I don't really understand why you implement this logic for the y-Label but not for the x-Label. I can now only type in the y input field and the button is enabled but when i trz to submit it tells me I can not because of the required in the InputTypeText component.
I think it makes sense to implement this logic either for both inputs or for none of them, as otherwise it can be confusing for the user. :)

Reply:
Done

Reply:
We implemented a new logic with a function called completedXandYLabels.